spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,6 +56,7 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
```

Comment:
I agree with @mkllnk , actually could be a good idea to mock the needed call to the class instead of changing the state with `#set()`.